### ISSUE TITLE
Crea una nueva aplicacion de index.py, tal vez gatos.py

### ISSUE THREAD

**Issue by akevalion** [jpiro80/Proyecto1#2]
un gasto
 * fecha del mes
 * agua
 * gas
 * luz
 * gastos comunes
 * internet
 * arriendo
 
idealmente se debe actualizar la tabla y el formulario para insertar nuevos gastos

**Comment by jpiro80** [comment]
Acá tuve un problema. Seguí paso a paso, línea por línea para ver si surgía algún error. Al escribir la línea 57 (aprox. min 42 del último video) que dice lo siguiente "self.tree.insert(' ', 0, text = row[1], values = row [2])" deberían aparecen en el cuadro las variables agregadas y los importes. Pero no aparecen. Dejo los archivos py y db para verificar dónde puede estar el error.

**Comment by akevalion** [comment]
De momento pude correr el programa 
![image](https://user-images.githubusercontent.com/10532890/121077515-b4b19400-c7a5-11eb-9526-bfe0011c941a.png)

Aunque yo esperaba un formulario, como por mes, es decir, que el formulario preguntara los precios de las cosas al mes

Nuevo pago del mes
Arriendo: ---------
Agua: -------
Gas: ------
Luz: ------
Gastos comunes: -------
Internet: --------

Algo de ese estilo en el formulario

sobre 

>  Al escribir la línea 57 (aprox. min 42 del último video) que dice lo siguiente "self.tree.insert(' ', 0, text = row[1], values = row [2])" deberían aparecen en el cuadro las variables agregadas y los importes. Pero no aparecen. 

Podemos revisar eso en algun meeting y verificar que todo este correcto

**Comment by jpiro80** [comment]
O sea, en lugar de "Agregar un nuevo impuesto" tendría que decir "Nuevo pago del mes" y solo ese cambio?

**Comment by akevalion** [comment]
Si pero pero con el formulario para agregar toda esa informacion 
```
Nuevo pago del mes
Arriendo: 180000
Agua: 10000
Gas: 7000
Luz: 17000
Gastos comunes: 80000
Internet: 14000
```
Algo de ese estilo en un formulario

**Comment by jpiro80** [comment]
Claro. Es justo en ese punto donde estoy trabado. Al parecer el formulario .db, es decir, el archivo hecho con el sqlite, debería poder insertarse en el cuadro con esos comandos. Especialmente con la línea 57 antes mencionada a la que el video hace referencia en el minuto 42. Pero esa es la parte que no sale.

**Comment by akevalion** [comment]
si deseas maniana podemos hacer un meeting y revisamos eso para que puedas seguir avanzando te parece tipo 6 pm para que lo agregue en el calendario

**Comment by jpiro80** [comment]
Milton. Como lo que me explicaste en la última conversación se me complicó demasiado, preferí volver atrás y hacer una tabla más simple. Por lo tanto voy a subir un nuevo index.py y un nuevo database.db para que veas cómo la diseñé. Aunque como viene pasando, no puedo hacer que aparezca la tabla en el cuadro. Yo empezaría primero por intentar resolver ese problema. Y una vez resuelto, podemos preocuparnos por modificar los nombres de las variables y complejizar la tabla agregando filas y columnas, te parece?

**Comment by akevalion** [comment]
dale, mantenlo simple, pero manten index.py y dabatase.db, como funcioanban en el video
